packages/react/src/KeybindingHint/KeybindingHint.examples.stories.tsx

```diff
@@ -20,7 +20,16 @@ export const PrimaryButton: StoryObj<KeybindingHintProps> = {
       Submit
     </Button>
   ),
-  args: {keys: 'Mod+Enter', variant: 'onEmphasis'},
+  args: {keys: 'Mod+Enter', variant: 'onPrimary'},
+}
+
+export const DangerButton: StoryObj<KeybindingHintProps> = {
```

Comment:
I didn't add an `onDanger` variant since danger buttons have a light resting background that works OK with the current design. Just added a story to demonstrate that